Unknown CMake command "cmake_parse_arguments" with cmake 2.8

When configuring the build (1.0.10) with cmake 2.8, I get the following error:
```
-- Using prebuilt re2c lexer
CMake Error at CMakeLists.txt:281 (cmake_parse_arguments):
  Unknown CMake command "cmake_parse_arguments".
Call Stack (most recent call first):
  CMakeLists.txt:311 (wabt_executable)
-- Configuring incomplete, errors occurred!
```

The following patch fixes this:
``` diff
diff -up wabt-1.0.10/CMakeLists.txt.orig wabt-1.0.10/CMakeLists.txt
--- wabt-1.0.10/CMakeLists.txt.orig     2019-03-07 01:41:26.000000000 +0100
+++ wabt-1.0.10/CMakeLists.txt  2019-03-15 10:43:39.046455124 +0100
@@ -277,6 +277,7 @@ if (NOT EMSCRIPTEN)
     endif ()
   endif ()
 
+  include(CMakeParseArguments)
   function(wabt_executable)
     cmake_parse_arguments(EXE "WITH_LIBM;INSTALL" "NAME" "SOURCES;LIBS" ${ARGN})
 
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -277,6 +277,7 @@ if (NOT EMSCRIPTEN)
     endif ()
   endif ()
 
+  include(CMakeParseArguments)
   function(wabt_executable)
     cmake_parse_arguments(EXE "WITH_LIBM;INSTALL" "NAME" "SOURCES;LIBS" ${ARGN})
 

